src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/GetAzureApplicationGatewayRedirectConfigurationCommand.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmApplicationGatewayRedirectConfiguration"),
+        OutputType(typeof(PSApplicationGatewayRedirectConfiguration), typeof(IEnumerable<PSApplicationGatewayRedirectConfiguration>))]
+    public class GetAzureApplicationGatewayRedirectConfigurationCommand : NetworkBaseCmdlet
```

Comment:
@akshaysngupta I am still unsure why this is being added as a cmdlet. Why can't a user just do the following:

```powershell
# Get all redirect configurations
$ApplicationGateway.RedirectConfigurations

# Get a specific redirect configuration
$RedirectConfigurationName = "foo"
$ApplicationGateway.RedirectConfigurations | where { $_.Name -eq $RedirectConfigurationName }
```

A new cmdlet should not be needed to get the property of an object.